src/keys/list.ts

```diff
@@ -31,14 +37,19 @@ import { EditorAction } from '../types'
 // TODO:: remove this command when quarterback start supporting list_item and the operation on the list
 export const skipCommandTracking =
   (command: Command) => (state: ManuscriptEditorState, dispatch?: Dispatch) => {
+    let handled = false // Variable to capture whether the command handled the action
+
     command(state, (tr) => {
+      handled = true // Set to true because the command handled the action and called dispatch
       if (dispatch) {
         skipTracking(tr)
         dispatch(tr)
       }
     })
```

Comment:
this should return result of command

Reply:
> skipTracking func will always return true
are you sure about that? it should return [transaction](https://github.com/Atypon-OpenSource/manuscripts-track-changes-plugin/blob/6e7d374c3396f34ed38b0a65322f00b8fdbab5ad/src/actions.ts#L74)

what return boolean is the [command](https://github.com/ProseMirror/prosemirror-state/blob/88e2a24104481098f0f660e74240c0f846e7fd6e/src/transaction.ts#L18) function